init/modules/EESSI/2023.06.lua

```diff
@@ -13,12 +13,24 @@ conflict("EESSI")
 -- this is a version-agnostic module file, works for EESSI/2023.06, EESSI/2025.06, etc.
 local eessi_version = myModuleVersion()
 local eessi_repo = "/cvmfs/software.eessi.io"
+local eessi_archdetect_prefix = pathJoin(eessi_repo, "versions", eessi_version, "init")
+-- for RISC-V clients we need to do some overrides, as things are stored in different CVMFS repositories
 if (subprocess("uname -m"):gsub("\n$","") == "riscv64") then
-    eessi_version = os.getenv("EESSI_VERSION_OVERRIDE") or "20240402"
-    eessi_repo = "/cvmfs/riscv.eessi.io"
-    LmodMessage("RISC-V architecture detected, but there is no RISC-V support yet in the production repository.\n" ..
-                "Automatically switching to version " .. eessi_version .. " of the RISC-V development repository " .. eessi_repo .. ".\n" ..
-                "For more details about this repository, see https://www.eessi.io/docs/repositories/riscv.eessi.io/.")
+    if (eessi_version == "2023.06") then
+        eessi_version = os.getenv("EESSI_VERSION_OVERRIDE") or "20240402"
+        eessi_repo = "/cvmfs/riscv.eessi.io"
+        LmodMessage("RISC-V architecture detected, but there is no RISC-V support yet in the production repository.\n" ..
+                    "Automatically switching to version " .. eessi_version .. " of the RISC-V development repository " .. eessi_repo .. ".\n" ..
+                    "For more details about this repository, see https://www.eessi.io/docs/repositories/riscv.eessi.io/.")
+    elseif (eessi_version == "2025.06") then
+        eessi_repo = "/cvmfs/dev.eessi.io/riscv"
```

Comment:
Should we also pick up on `$EESSI_VERSION_OVERRIDE` here, something like:

```suggestion
        eessi_repo = "/cvmfs/dev.eessi.io/riscv"
        eessi_version = os.getenv("EESSI_VERSION_OVERRIDE") or eessi_version
```

I imagine for RISC-V we'll have "iterations" in `dev.eessi.io`, like `2025.06-202509` or something